templates/PAAStack.yaml

```diff
@@ -63,11 +67,74 @@ Resources:
         Ref: vendorId
     UpdateReplacePolicy: Delete
     DeletionPolicy: Delete
+  CrlBucketName6B0FB645:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Delete
+    Properties:
+      BucketName:
```

Comment:
doesn't look like you generated these files using the generateCfn command (so they deviate a little bit). Please sync up